src/components/SidebarComponent.js

```diff
@@ -0,0 +1,17 @@
+import React from 'react';
+import styles from '../assets/stylesheets/sidebar.module.css';
+
+function Sidebar() {
+  return (
+    <div className={styles.sidebar}>
+      <ul>
+        <li>Home</li>
+        <li>About</li>
+        <li>Services</li>
+        <li>Contact</li>
+      </ul>
```

Comment:
Please adapt this left sidebar menu to the required options given in the project's requirements. Point 2) In the navigation panel, the user can see links to:
Motorcycles/doctors/classes/items that you selected as a theme. In our case: **Coworking Spaces**
**"Reserve"** form.
**"My reservations"**.
"**Add coworking space** motorcycle/doctor/class/item that you selected as a theme" (the link is visible to everybody).
"**Delete coworking space** motorcycle/doctor/class/item that you selected as a theme" (the link is visible to everybody).
We can check it together on a meeting, to define it completely. Thank you.